ccd-definition/CaseEventToFields/CareSupervision/manageOrders/1_initialSection.json

```diff
@@ -3,13 +3,25 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "manageOrders",
-    "CaseFieldID": "manageOrdersOperation",
+    "CaseFieldID": "orderTempQuestions",
```

Comment:
just double checking but is this change okay for the closed state version of the event

Reply:
good point, double checking with business if any upload available for closed state

Reply:
nice one, doublechecked with Gail, I've restricted the dropdown to only have "Create" in a close state